include/boost/json/monotonic_resource.hpp

```diff
@@ -13,30 +14,97 @@
 #include <boost/json/detail/config.hpp>
 #include <boost/json/storage_ptr.hpp>
 
+#include <limits>
+#include <new>
+
+#ifndef BOOST_JSON_MAX_ALIGN
+# ifdef BOOST_JSON_STANDALONE
+#  define BOOST_JSON_MAX_ALIGN alignof(std::max_align_t)
+# else
+#  define BOOST_JSON_MAX_ALIGN boost::move_detail::alignment_of<boost::move_detail::max_align_t>::value
```

Comment:
This adds a dependency on Boost.Move? Why is it accessing a detail namespace? That's bad form.